game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/game/CarMapGenerator.scala

```diff
@@ -65,10 +65,10 @@ class CarMapGenerator(seed: Int) extends GameMapGenerator {
 
         val mapObjects = mutable.SortedMap[Double, Int]()
         mapObjects.put(0.0, Config.EMPTY_MAP_OBJECT)
-        mapObjects.put(Config.MUD_GENERATION_PERCENTAGE * 0.01 * 0.35, Config.MUD_MAP_OBJECT)
+        mapObjects.put(Config.MUD_GENERATION_PERCENTAGE * 0.01 * 0.47, Config.MUD_MAP_OBJECT)
         mapObjects.put(mapObjects.lastKey + Config.BOOST_GENERATION_PERCENTAGE * 0.01, Config.BOOST_MAP_OBJECT)
         mapObjects.put(mapObjects.lastKey + Config.OIL_ITEM_GENERATION_PERCENTAGE * 0.01, Config.OIL_ITEM_MAP_OBJECT)
-        mapObjects.put(mapObjects.lastKey + Config.WALL_GENERATION_PERCENTAGE * 0.01, Config.WALL_MAP_OBJECT)
+        mapObjects.put(mapObjects.lastKey + Config.WALL_GENERATION_PERCENTAGE * 0.01 * 1.3, Config.WALL_MAP_OBJECT)
```

Comment:
What are these calibration constants for exactly? Shouldn't we only modify the config values for this

Reply:
it's to offset the walls that get removed in the post processing to ensure there is always a path of at least 15 blocks